src/schema.ts

```diff
@@ -314,6 +314,46 @@ export const applicationRelations = relations(applications, ({ many }) => ({
   accessTokens: many(accessTokens),
 }));
 
+export const accessGrants = pgTable(
+  "access_grants",
+  {
+    id: uuid("id").$type<Uuid>().primaryKey(),
+    token: text("token").notNull().unique(),
+    expiresIn: integer("expires_in").notNull(),
+    redirectUri: text("redirect_uri").notNull(),
+    scopes: scopeEnum("scopes").array().notNull(),
+    applicationId: uuid("application_id")
+      .$type<Uuid>()
+      .notNull()
+      .references(() => applications.id, { onDelete: "cascade" }),
+    resourceOwnerId: uuid("resource_owner_id")
+      .$type<Uuid>()
+      .notNull()
+      .references(() => accountOwners.id, { onDelete: "cascade" }),
+    codeChallenge: text("code_challenge"),
+    codeChallengeMethod: text("code_challenge_method"),
+    createdAt: timestamp("created_at", { withTimezone: true })
+      .notNull()
+      .default(currentTimestamp),
+    revokedAt: timestamp("revoked_at", { withTimezone: true }),
```

Comment:
It's trivial, but could you rename them to `created` and `revoked` for consistency?